CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+Storybook 7 is now officially supported!
+
+- [breaking] Drop Storybook 6 support
```

Comment:
Technically these changes will still actually work with Storybook v6. But I'm not going to test on that version, and figured I'll just say we only support Storybook 7.